topics/variant-analysis/tutorials/sars-cov-2/tutorial.md

```diff
@@ -67,14 +93,14 @@ SRA can be reached either directly through it's website, or through the tool pan
 
 > ### {% icon comment %} Comment
 >
-> Initially the tool panel option exists only on the [usegalaxy.org server](https://usegalaxy.org/).  Support for the direct connection to SRA will be included in the 20.05 release of Galaxy
+> Initially the tool panel option for accessing SRA exists only on the [usegalaxy.org server](https://usegalaxy.org/).  Support for the direct connection to SRA will be included in the 20.05 release of Galaxy
 {: .comment}
 
 
 > ### {% icon hands_on %} Hands-on: Explore SRA Entrez
 >
-> 1. Go to [usegalaxy.org](https://usegalaxy.org/)
-> 1. If your history is not already empty, than start a new history (see here for more on Galaxy histories)
+> 1. Go to your Galaxy instance of choice such as one of the [usegalaxy.org](https://usegalaxy.org/), [usegalaxy.eu](https://usegalaxy.eu), [usegalaxy.org.au](https://usegalaxy.org.au) or any other. (This tutorial uses usegalaxy.org).
+> 1. If your history is not already empty, than start a new history (see [here](https://training.galaxyproject.org/training-material/topics/galaxy-interface/tutorials/history/tutorial.html) for more on Galaxy histories)
```

Comment:
I'd use the `create_new_history` snippet here instead.

Reply:
Hmmm ... my tutorial is in `introductions`. I did not mean to touch the `topics/variant-analysis/tutorials/sars-cov-2/tutorial.md` version

Reply:
This portion is commented out for now